index.html

```diff
@@ -122,288 +122,79 @@ <h2 class="item2">Multi-Post Stories Gain+Glory</h2>
-      <p class="sub-head">Skills</p>
-     <ul class="flex3">
-     <li class="lang">Database Management</li>
-     <li class="lang">Version Control</li>
-     </ul>
-     <ul class="flex4">
-       <li class="lang">CLI</li>
-       <li class="lang">Web development</li>
-       <li class="lang">API Design</li>
-     </ul>
-   </div>
-    </div>
-    <div class="flex5">
+            <p class="sub-head">Languages</p>
+             <ul class="flex2">
+              <li class="lang">JavaScript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+             </ul>
+       </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/Icon.png">
+          <p class="sub-head">Frameworks</p>
+            <ul class="flex3">
+             <li class="lang">React.js</li>
+             <li class="lang">Ruby on Rails</li>
+             <li class="lang">RSpec</li>
+           </ul>
+            <ul class="flex4">
+              <li class="lang">Capybara</li>
+              <li class="lang">Selenium</li>
+           </ul>
+        </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/ico.png">
+          <p class="sub-head">Skills</p>
+           <ul class="flex3">
+            <li class="lang">Database Management</li>
+            <li class="lang">Version Control</li>
+           </ul>
+          <ul class="flex4">
+           <li class="lang">CLI</li>
+           <li class="lang">Web development</li>
+           <li class="lang">API Design</li>
+         </ul>
+       </div>
+     </div>
+      <div class="flex5">
         <p id="bio">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
        </div>
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    </section>
+   <section>
+     <form action="https://formspree.io/f/xoqzonzv" method="post" class="form">
+       <div class="flex6">
+         <input type="text" name="name" placeholder="Full name" maxlength="30" class="text-placeholder" required>
+       </div>
+        <div class="flex6">
+          <input type="email" name="email" placeholder="Email address" class="text-placeholder" required>
+        </div>
+        <div class="flex7">
+         <textarea name="message" placeholder="Hey I’d like to connect to..." maxlength="500" class="style-placeholder"></textarea>
+         <input type="submit" name="submit" class="touch" value="Get in touch">
+        </div> 
```

Comment:
- [ ] As required for the project, kindly ensure that your design sticks as much as possible to the one form your chosen Figma template. Below you can the difference.

| Your Design | Figma Design |
| --------------------- | ----------------- |
| ![Screenshot from 2023-06-07 11-11-48](https://github.com/Bupilipili/MY-PORTFOLIO/assets/83042742/d3880111-b224-4c9a-8acb-52d746ae5ab1) | ![Screenshot from 2023-06-07 11-13-10](https://github.com/Bupilipili/MY-PORTFOLIO/assets/83042742/40ba6b80-e1c9-4809-a24a-76afb7e0085f) |